src/Files.App/Views/Layouts/GridLayoutPage.xaml

```diff
@@ -1007,7 +1007,6 @@
 					x:Name="FileList"
 					Padding="8"
 					VerticalContentAlignment="Stretch"
-					tui:ScrollViewerExtensions.EnableMiddleClickScrolling="{x:Bind IsMiddleClickToScrollEnabled, Mode=OneWay}"
```

Comment:
Was this removed intentionally?

Reply:
We don't have `EnableMiddleClickScrolling` support in CommunityToolkit 8.x

Reply:
Is there an issue tracking this?

Reply:
There's one in https://github.com/CommunityToolkit/Windows/issues/426.
I think we can port it manually after this PR being merged. The PR has been very large, and I don't want to add even more things.

Reply:
Fair enough